arabic/dream/another/It was a nice dream.md

```diff
@@ -0,0 +1,2 @@
+كنت أحاول الفرار من المجرمين وبينما أركض قلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت فواجهتهم حتى قضيت على أربعة منهم وفر الباقى قبل أن ينادينى أحدهم لأسرع لأن ميعاد المدرسة سيفوتنى ان لم أستيقظ  
+[nice dream](../dream/dream.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.